ui/src/app/models/dagRunDetail.model.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class DagRunDetailModel {
```

Comment:
This class should be called `JobInstanceModel` imo, because the backend returns instances of `JobInstance`. Also, the `dagRun.model.ts`, `dagRunSearchResponse.model.ts` etc. should probably go in a subfolder called `dagRuns` to replicate the folder structure in the backend code.

Reply:
Done